src/site/content/en/learn/css/box-model/index.md

```diff
@@ -0,0 +1,321 @@
+specific job for us.
+
+![The four main sections of the box model: content box, padding box,
+border box and margin box](https://assets.codepen.io/174183/bm-1.jpg)
+
+You start with **content box**, which is the layer that our content
+lives in. As you learned before: this content can control the size of
+its parent, so is usually the most variably sized layer.
+
+The **padding box** surrounds the **content box** and is where the
+space created by the [`padding`
+property](https://developer.mozilla.org/en-US/docs/Web/CSS/padding).
+Because padding is **inside the box**, the background of the box will
+be visible in the space that it creates. If our box has overflow rules
+set, such as `overflow: auto` or `overflow: scroll` set, the
+scrollbars will occupy this space too.
+
+<iframe height="500" style="width: 100%;" scrolling="no" title="Learn CSS - Box model intro" src="https://codepen.io/argyleink/embed/bGNmgGW?height=500&theme-id=38982&default-tab=result" frameborder="no" loading="lazy" allowtransparency="true" allowfullscreen="true">
+  See the Pen <a href='https://codepen.io/argyleink/pen/bGNmgGW'>Learn CSS - Box model intro</a> by Piccalilli
+  (<a href='https://codepen.io/piccalilli'>@piccalilli</a>) on <a href='https://codepen.io'>CodePen</a>.
+</iframe>
+
+The **border box** surrounds the **padding box** and its space is
+occupied by the `border` value. The **border box** is the bounds of
+your box and the **border edge** is the limit of what you can visually
+see. The [`border`
+property](https://developer.mozilla.org/en-US/docs/Web/CSS/border) is
+used to visually frame an element.
+
+The final layer, the **margin box**, is the space around your box,
+defined by the `margin` rule on your box, or by parent-defined space,
+[such as `gap`](https://developer.mozilla.org/en-us/docs/Web/CSS/gap).
+Properties such as
+[`outline`](https://developer.mozilla.org/en-US/docs/Web/CSS/outline)
+and and
+[`box-shadow`](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow)
+occupy this space too because they are **painted**, so they don't
+affect the size of our box. You could have an `outline-width` of
+`200px` on our box and everything **inside and including the border
+box would be exactly the same size**.
+
+<iframe height="500" style="width: 100%;" scrolling="no" title="Learn CSS - Box model intro" src="https://codepen.io/piccalilli/embed/3a29b4a7d35ef7e1bd205be9ac65e25e?height=500&theme-id=38982&default-tab=result" frameborder="no" loading="lazy" allowtransparency="true" allowfullscreen="true">
+  See the Pen <a href='https://codepen.io/piccalilli/pen/3a29b4a7d35ef7e1bd205be9ac65e25e'>Learn CSS - Box model intro</a> by Piccalilli
+  (<a href='https://codepen.io/piccalilli'>@piccalilli</a>) on <a href='https://codepen.io'>CodePen</a>.
+</iframe>
+
+## A useful analogy
+
+The box model is complex to understand, so let's recap what we've
+learned with an analogy.
+
+![A group of photo frames, labelled with box model
```

Comment:
This should probably use our image CDN / `{% Img %}` shortcode but that means you need a way to upload it... Let me talk to one of my teammates to see if we can create an account for you.

Reply:
That would be ace because I will 100% forget to fix these otherwise :)